src/main/graphql/GetConsignmentSeriesAndBody.graphql

```diff
@@ -0,0 +1,10 @@
+query getConsignmentSeriesAndBody($consignmentId: UUID!) {
```

Comment:
Maybe `getConsignmentSummary`? We'll need to add other fields to this query like the consignment reference, so I think it's worth making the name more general.